src/utils/AppError.ts

```
@@ -0,0 +1,13 @@
+export class AppError extends Error {
```

You brought the AppError class from our express app here. So we can have a base error class!. I think it’s cool, I didn’t do this cause I was gunning for the approach where we define the kind of error we wish to throw. We could extend error and call our class InvalidIdError, then maybe throw InvalidIdError when we need it. But I guess this works